src/components/Header/authApiMockResponse.ts

```diff
@@ -0,0 +1,106 @@
+export const mockResponseData = {
```

Comment:
Add a comment above saying these three variables are mock responses from the `platform.nypl.org` API.

Reply:
You can also include somehow that the URL endpoint is set in  the `patronApiUrl` variable.